include/deal.II/lac/cuda_vector.h

```diff
@@ -214,9 +216,6 @@ namespace LinearAlgebra
        * calling separate methods means to load the calling vector @p this
        * twice. Since most vector operations are memory transfer limited, this
        * reduces the time by 25\% (or 50\% if @p W equals @p this).
-       *
-       * For complex-valued vectors, the scalar product in the second step is implemented as
-       * $\left<v,w\right>=\sum_i v_i \bar{w_i}$.
```

Comment:
Could I advocate for leaving this sentence in anyway? I don't know if we're ever going to get around to implementing complex values in CUDA, but it'd be a shame to have to go through things and figure out what each function is supposed to do by comparing with the other vector classes.

Reply:
👍

Reply:
I have put it back.